### DISCUSSION TITLE
Minimum libcurl version

### DISCUSSION THREAD

**Discussion by skidoo23** [MusicPlayerDaemon/MPD#1716] [General]
MPD commit 4efd0a9f77d2ee713443f9b61fb7828af0eb4c42 makes use of CURLINFO_CONTENT_LENGTH_DOWNLOAD_T.

According to MPD/src/lib/curl/meson.build project requires libcurl 7.33 as minimum which is about nine years old. But  CURLINFO_CONTENT_LENGTH_DOWNLOAD_T was introduced with libcurl 7.55 about five years ago. Should MPD raise its minimum libcurl version to 7.55 or do some magic to maintain compatible to older versions?

**Comment by MaxKellermann** [comment]
Raise minimum.
I changed to the new option because the old one emitted a deprecation warning with new curl versions. But after so many years, it's reasonable to require a newer version.
